individual-code-of-conduct.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+menu_id: download
+title: Code of Conduct
+---
+
+# Presto Code of Conduct
+
+Like the technical community as a whole, the Presto team and community is made up of a mixture of professionals and
+volunteers from all over the world, working on every aspect of Presto.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we 
+have a few ground rules that we ask people to adhere to. This code applies equally to all participants in the Presto
+project.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a 
+guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the Presto project or Presto Software Foundation. This includes 
+Slack, the mailing lists, the issue tracker, PSF events, and any other forums created by the project team which the
+community uses for communication. In addition, violations of this code outside these spaces may affect a person's
+ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing `conduct@prestosql.io`. 
+-- For more details please see our Reporting Guidelines
```

Comment:
This renders as EN DASH, should be EM DASH. Maybe remove the line break before `--`

Reply:
"Reporting Guidelines" should be a link